helm-chart/kuberay-operator/README.md

```diff
@@ -18,13 +18,14 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 To avoid duplicate CRD definitions in this repo, we reuse CRD config in `ray-operator`:
 ```console
-$ kubectl apply -k "../../ray-operator/config/crd"
+$ kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0"
```

Comment:
`ref=v0.3.0` is something I am confusing, current HEAD TAG is v0.2.0. Before we cut v0.3.0, I think this is invalid?

Reply:
0.3.0 was intentional here and below, in preparation of the upcoming tag that will support this functionality. I guess you could also point this one to the default branch and the one below to `latest` (with some dirty ol' grepping or so ref https://stackoverflow.com/q/24987542), but for helm related stuff, immutable tags are generally recommended